litho-it/build.gradle

```diff
@@ -89,30 +89,24 @@ dependencies {
     testImplementation deps.powermockReflect
     testImplementation deps.khronos
     testImplementation files(Jvm.current().getToolsJar())
-    testImplementation files(org.gradle.internal.jvm.Jvm.current().getJre().homeDir.getAbsolutePath() + '/lib/rt.jar')
+    testImplementation files(Jvm.current().getJre().homeDir.getAbsolutePath() + '/lib/rt.jar')
 }
 
-// Source - https://github.com/nenick/AndroidStudioAndRobolectric/blob/master/app/build.workaround-missing-resource.gradle
-// Workaround for missing test resources when running unit tests within android studio.
-// This copies the test resources next to the test classes for each variant.
-// Tracked at https://github.com/nenick/AndroidStudioAndRobolectric/issues/7
-// Original solution comes from https://code.google.com/p/android/issues/detail?id=136013#c10
-// See also https://code.google.com/p/android/issues/detail?id=64887
-
 gradle.projectsEvaluated {
-  // Base path which is recognized by android studio.
-  def testClassesPath = "${buildDir}/intermediates/classes/test/"
-  // Copy must be done for each variant.
-  def variants = android.libraryVariants.collect()
-
-  variants.each { variant ->
-    def variationName = variant.name.capitalize()
-    def variationPath = variant.buildType.name
-
-    // Specific copy task for each variant
-    def copyTestResourcesTask = project.tasks.create("copyTest${variationName}Resources", Copy)
-    copyTestResourcesTask.from("${projectDir}/src/test/resources")
-    copyTestResourcesTask.into("${testClassesPath}/${variationPath}")
-    copyTestResourcesTask.execute()
-  }
+    // Base path which is recognized by android studio.
+    def testClassesPath = "${buildDir}/intermediates/classes/test/"
+    // Copy must be done for each variant.
+    def variants = android.libraryVariants.collect()
+
+    variants.each { variant ->
+        def variationName = variant.name.capitalize()
+        def variationPath = variant.buildType.name
+
+        task "copyTest${variationName}Resources"(type: Copy) {
```

Comment:
Ahh, that's how you do this now. Thanks for looking into this.

Reply:
@passy `Copy` was always there! It's just that now you need to work the task execution through Task dependencies, or utilize Worker API through a custom plugin that would host the task (which would be overkill in that case IMO). Since we need the `test` task to always run after those tasks have executed, we just add a dependency on it, for the created tasks.

Reply:
I didn't realise you could just create an ad-hoc task like this and depend on it. If I remember correctly, they removed the `.execute()` method on tasks you create imperatively, didn't they?

Reply:
Yeap, it's removed and the project does not build with it. This Groovy syntax has the same effect as old .create() but registers it. In KotlinDSL we would explicitly register it and then configure it!